src/components/Session/EffectCard.jsx

```diff
@@ -0,0 +1,151 @@
+/*
```

Comment:
Here again, the effect card is un-necessary. Better to name it as a "TxHistoryRow". Card implies that it is a "card" like thing with many different uses.